src/components/PrivacySupportSidebar.tsx

```
@@ -1,33 +1,17 @@
-  ArrowTopRightOnSquareIcon,
-  UserIcon
+  ArrowTopRightOnSquareIcon
 } from '@heroicons/react/24/outline';
 
 interface PrivacySupportSidebarProps {
   className?: string;
-  onRequestConsultation?: () => void;
 }
 
-const PrivacySupportSidebar = ({ onRequestConsultation }: PrivacySupportSidebarProps) => {
+const PrivacySupportSidebar = ({}: PrivacySupportSidebarProps) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)